docs/samples/contact-center/app.js

```diff
@@ -94,7 +99,7 @@ function initWebex(e) {
     console.log('Authentication#initWebex() :: Webex Ready');
 
     authStatusElm.innerText = 'Saved access token!';
-
+    registerStatus.innerHTML = 'Not Registered';
```

Comment:
Let's call it it Not Subscribed and this text should update once we receive welcome event

Reply:
addressed